packages/react/src/themes/teams/components/List/listStyles.ts

```diff
@@ -3,18 +3,15 @@ import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
 import { ListProps } from '../../../../components/List/List'
 
 const listStyles: ComponentSlotStylesInput<ListProps> = {
-  root: ({ props }): ICSSInJSStyle => {
-    const { as, debug } = props
-    return {
-      ...(debug && debugRoot()),
-      ...((as === 'ul' || as === 'ol') && {
-        listStyle: 'none',
-        display: 'block',
-        padding: 0,
-        margin: 0,
-      }),
-    }
-  },
+  root: ({ props: p }): ICSSInJSStyle => ({
+    ...(p.debug && debugRoot()),
+    display: p.vertical ? 'block' : 'flex',
```

Comment:
What's the reasoning on horizontal lists being block level elements vs inline elements (`inline-flex`)? What's use cases are we targeting?

Reply:
No strong reasoning for this, I was just following the paradigm we had before. `inline-flex` works as expected as well, and may be even better to the usecases we have, something like this:
![image](https://user-images.githubusercontent.com/4512430/62117333-080a4f80-b2bc-11e9-9f08-170187ce5b60.png)

Will change it